.github/workflows/release-app.yml

```diff
@@ -122,7 +122,7 @@ jobs:
             fi
             sleep 10
           done
-          echo "runner_type=windows-latest" >> $GITHUB_OUTPUT
```

Comment:
our self hosted runner has priority and it's a windows 10 or 11 dont remember (computer at home)